src/backend/catalog/namespace.c

```diff
@@ -3099,6 +3440,61 @@ get_namespace_oid(const char *nspname, bool missing_ok)
 	return oid;
 }
 
+/*
+ * get_module_oid - given a (possibly fully qualified) module name, look up the OID
+ *
+ * If missing_ok is false, throw an error if namespace or module name not found.  If
+ * true, just return InvalidOid.
+ */
+Oid
+get_module_oid(List *names, bool missing_ok)
+{
+	char	   *schemaname;
+	char	   *modulename;
+	char       *objname;
+	Oid			moduleId = 0;
```

Comment:
ok I'll set it to InvalidOid further down, because at some point that will need to be done, but yes it can happen close to the point where we don't find a valid module.